### PR TITLE
Update the order of remarks and examples to align with docs.ms

### PR DESCRIPTION
**Pull request by sima-zhu**
We have different definition for the order of `<remarks>` and `<examples>`.
Update our template to align with mdoc.

[Link](https://ceapex.visualstudio.com/Engineering/_git/docs-ui?path=/packages/docfx-templates/partials/dotnet/member.tmpl.partial&version=GBmaster&line=14&lineEnd=22&lineStartColumn=1&lineEndColumn=13&lineStyle=plain&_a=contents)

In github io: 
image

In Docs.ms:
image

### CODE REVIEW

**Review by benbp — APPROVED** [review]

**Review by weshaggard — APPROVED** [review]

**Comment by weshaggard** [comment]
@sima-zhu lets wait until after releases to merge this as it isn't very important and we want to avoid any issues with release week.

**Pull request merged**
